RecoHI/HiTracking/python/HILowPtConformalPixelTracks_cfi.py

```diff
@@ -124,25 +124,23 @@
     SeedingHitSets = "hiConformalPixelTracksPhase1HitQuadrupletsCA",
 )
 
-
-
-
-hiConformalPixelTracksSequence = cms.Sequence(
-    hiTrackingRegionWithVertex +
-    hiConformalPixelTracksHitDoublets +
-    hiConformalPixelTracksHitTriplets +
-    pixelFitterByConformalMappingAndLine +
-    hiConformalPixelFilter +
+hiConformalPixelTracksTask = cms.Task(
+    hiTrackingRegionWithVertex ,
+    hiConformalPixelTracksHitDoublets ,
+    hiConformalPixelTracksHitTriplets ,
+    pixelFitterByConformalMappingAndLine ,
+    hiConformalPixelFilter ,
     hiConformalPixelTracks
 )
-
-hiConformalPixelTracksSequencePhase1 = cms.Sequence(
-    hiConformalPixelTracksPhase1TrackingRegions +
-    hiConformalPixelTracksPhase1SeedLayers +
-    hiConformalPixelTracksPhase1HitDoubletsCA +
-    hiConformalPixelTracksPhase1HitQuadrupletsCA +
-    pixelFitterByConformalMappingAndLine +
-    hiConformalPixelTracksPhase1Filter +
+hiConformalPixelTracksSequence = cms.Sequence(hiConformalPixelTracksTask)
```

Comment:
Since this was only needed later in RecoHI/HiTracking/python/hiMergedConformalPixelTracking_cff.py, the sequence can be removed from here